app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.LoginActivity">
+
+    <EditText
+        android:id="@+id/text_usermail"
+        android:layout_width="0dp"
```

Comment:
For the dimensions we use the `dimens.xml`. Here you can read about this: https://medium.com/@studymongolian/using-dimens-xml-in-android-10dec2fe485c. Change it everywhere, please.